frontend/app/routes/_public+/apply+/$id+/application-delegate.tsx

```diff
@@ -0,0 +1,67 @@
+import { LoaderFunctionArgs, json } from '@remix-run/node';
+import { useLoaderData } from '@remix-run/react';
+
+import { faChevronLeft } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Trans, useTranslation } from 'react-i18next';
+
+import { ButtonLink } from '~/components/buttons';
+import { InlineLink } from '~/components/inline-link';
+import { getApplyFlow } from '~/routes-flow/apply-flow';
+import { getTypedI18nNamespaces } from '~/utils/locale-utils';
+import { RouteHandleData } from '~/utils/route-utils';
+
+const i18nNamespaces = getTypedI18nNamespaces('eligibility');
+
+export const handle = {
+  breadcrumbs: [
+    // prettier-ignore
+    { labelI18nKey: 'eligibility:breadcrumbs.canada-ca', to: '/personal-information' },
+    { labelI18nKey: 'eligibility:breadcrumbs.benefits' },
+    { labelI18nKey: 'eligibility:breadcrumbs.dental-coverage' },
+    { labelI18nKey: 'eligibility:breadcrumbs.canadian-dental-care-plan' },
+  ],
+  i18nNamespaces,
+  pageIdentifier: 'CDCP-00XX',
+  pageTitleI18nKey: 'eligibility:application-delegate.page-title',
+} as const satisfies RouteHandleData;
+
+export async function loader({ request, params }: LoaderFunctionArgs) {
+  const applyFlow = getApplyFlow();
+  const { id } = await applyFlow.loadState({ request, params });
+
+  return json({ id });
+}
+
+export default function ApplyFlowApplicationDelegate() {
+  const { id } = useLoaderData<typeof loader>();
+  const { t } = useTranslation(i18nNamespaces);
+
+  const contactservicecanada = <InlineLink to={t('application-delegate.contact-service-canada-href')} />;
+  const preparingtoapply = <InlineLink to={t('application-delegate.preparing-to-apply-href')} />;
+
+  return (
+    <>
+      <br />
+      <div className="max-w-prose">
+        <p>
+          <Trans ns={i18nNamespaces} i18nKey="application-delegate.description1" components={{ contactservicecanada }} />
+        </p>
+        <br />
+        <p>
+          <Trans ns={i18nNamespaces} i18nKey="application-delegate.description2" components={{ preparingtoapply }} />
+        </p>
+        <br />
```

Comment:
This should be removed and replaced with styling (margin or padding).

Reply:
Removed BR and used mt/mb margins